verifiers/internal/gcb/testdata/gcloud-container-invalid-slsaheader.json

```diff
@@ -0,0 +1,94 @@
+{
+    "image_summary": {
+      "digest": "sha256:1a033b002f89ed2b8ea733162497fb70f1a4049a7f8602d6a33682b4ad9921fd",
+      "fully_qualified_digest": "us-west2-docker.pkg.dev/gosst-scare-sandbox/quickstart-docker-repo/quickstart-image@sha256:1a033b002f89ed2b8ea733162497fb70f1a4049a7f8602d6a33682b4ad9921fd",
+      "registry": "us-west2-docker.pkg.dev",
+      "repository": "quickstart-docker-repo"
+    },
+    "provenance_summary": {
+      "provenance": [
+        {
+          "build": {
+            "intotoStatement": {
+              "_type": "https://in-toto.io/Statement/v0.1",
+              "predicateType": "https://slsa.dev/provenance/v0.1",
+              "slsaProvenance": {
+                "builder": {
+                  "id": "https://cloudbuild.googleapis.com/GoogleHostedWorker@v0.2"
+                },
+                "materials": [
+                  {
+                    "uri": "https://github.com/laurentsimon/gcb-tests/commit/fbbb98765e85ad464302dc5977968104d36e455e"
```

Comment:
will this always be a github source? I guess not. That is a little annoying: should we open our `--source` to handle other source material?

Reply:
It won't always be a GitHub source. `-source` is not restricted to github, is it?

Reply:
GCB definitely supports native Cloud Source Repositories

Reply:
OK I think I mixed this up: for gha verifier we always expect github.com URIs, but for GCB in `VerifySourceURI`we just expected some URI

I think this is fine then?

Reply:
Yes that's correct. I'll double-check when I update this PR. (Leaving this conversation unresolved till then so that I don't forget)